encoding/wkt/decode.go

```diff
@@ -163,18 +163,20 @@ func createGeomCollectionForWkt(wkt string) (*geom.GeometryCollection, error) {
 	return gc, nil
 }
 
-func readCoordsDim1(l geom.Layout, wkt string) ([]geom.Coord, string, error) {
-	isEmpty := strings.HasSuffix(wkt, tEmpty)
-	if isEmpty {
+func readCoordsDim1(l geom.Layout, wkt string, allowEmpty bool) ([]geom.Coord, string, error) {
```

Comment:
nit: i think `allowEmpty` is a bit misleading, as we allow empty just below here. maybe "allowEmptyComponentsInBraceContent" or something?

Reply:
Sure, renamed it `allowEmptyCoordInBrace`